discovery-provider/plugins/notifications/src/main.ts

```diff
@@ -62,14 +61,12 @@ export class Processor {
     // process events
     logger.info('processing events')
     this.isRunning = true
-    // NOTE: Temp for testing
-    // TODO remove
     const redis = await getRedisConnection()
     await redis.set(config.lastIndexedMessageRedisKey, new Date(Date.now()).toISOString())
     await redis.set(config.lastIndexedReactionRedisKey, new Date(Date.now()).toISOString())
     while (this.isRunning) {
-      // NOTE: Temp to stop app notifiations
-      // await sendAppNotifications(this.listener, this.appNotificationsProcessor)
+      // Comment out to prevent app notifications until complete
+      await sendAppNotifications(this.listener, this.appNotificationsProcessor)
```

Comment:
does this need to be commented out as the comment above says?

Reply:
it does not unless we are pushing to stage
This was only commented out so that michelle could push out her changes without pushing app notifications which were not ready